cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -46,6 +46,9 @@ type indexArgs struct {
 	// Branch is the branch name.
 	Branch string
 
+	// Branches to index.
+	Branches []zoekt.RepositoryBranch
```

Comment:
Since we will pass this info directly to gitIndex, should we simply use []string instead?